File: 3302.py
n = int(input())
e = []
for c in range(0, n):
    c = input()
    e += [c]
for d in range(0, n):
    if d == 0:
        print(f'resposta {d+1}: {e[d]}')
    else:
        print(f'resposta {d+1}: {e[d]}')

File: 3302.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var n = parseInt(lines.shift());
let e = []
for(var c = 0; c < n; c++){
    var ca = lines.shift();
    e.push(ca);}
for(var d = 0; d < n;d++){
    if(d === 0){
        console.log(`resposta ${d+1}: ${e[d]}`)}
    else{
        console.log(`resposta ${d+1}: ${e[d]}`)}}
